.circleci/config.yml

```diff
@@ -38,32 +36,38 @@ commands:
       #- run: sudo apt-get install postgresql-client
       - run:
           name: Set up database
-          command: bundle exec rake db:setup
+          command: bundle exec rake db:create db:schema:load
 
 jobs:
-  install_dependencies:
-    executor: rails
-    steps:
-      - checkout
-      - install-dependencies
-  static_analysis:
+  load_data:
     executor: rails
     steps:
       - checkout
-      - install-dependencies
+      - ruby/install-deps
       - db-setup
-      - ruby/rubocop-check
+      - run:
+          # Ensure seeds run without issues
+          name: Load seeds
+          command: bin/rails db:seed
       - run:
           # Ensure sample_data runs without issues
-          name: Seed with sample data
+          name: Load sample data
           command: bin/rails db:sample_data
+  rubocop:
+    executor: rails
+    steps:
+      - checkout
+      - ruby/install-deps
+      - ruby/rubocop-check
   rspec:
     executor: rails
+    parallelism: 4
```

Comment:
Is there a secret rationale to this number?

Reply:
Not really. Circle's [example Rails project](https://github.com/CircleCI-Public/circleci-demo-ruby-rails/blob/2033c5f8da3010941127e0f9133925bd8efc7a79/.circleci/config.yml#L19) suggests `3` without any explanation. Another consideration is that the free tier allows parallelism of up to 30 (I think), so you could theoretically dial this up to 30, but then you'd be burning through tons of CPU credits, and you'd potentially starve yourself of resources if your team wanted to building multiple PRs at once. I chose `4` as a compromise to get a non-trivial amount of parallelism but not hog resources.

In practice, you probably won't see a benefit until you have a large number of slow specs. But I think there is value to including it in our baseline config so that people are aware that the setting exists and can be tuned.

Reply:
Hmm. I just tested and it is faster without the parallelism. 😁 I guess it makes sense that it adds some overhead and you only benefit if you have a sufficient number of tests to make it worth it.

I'll leave `parallelism: 4` in there but commented out, so that at least people are aware of the setting and can opt in if/when their project reaches a certain size.